### DISCUSSION TITLE
LBX with Steam Games

### DISCUSSION THREAD

**Discussion by Gator-RR** [JonnyHaystack/HayBox#38] [General]
Basically it will work with Steam games on LBX, but only if I reconfigure it every time, in the Steam Controller settings. Anyone else had better luck? Its just very inconsistent for me. Sometimes it reads as an XBox controller (usually the C-stick Y axis won't work these times), other times as "Generic" which works for steam games (Rivals of Aether and Rushdown Revolt) but only until it gets unplugged. No idea what I am doing wrong, but it is maddening.

**Comment by Animach1ne** [comment]
> Basically it will work with Steam games on LBX, but only if I reconfigure it every time, in the Steam Controller settings. Anyone else had better luck? Its just very inconsistent for me. Sometimes it reads as an XBox controller (usually the C-stick Y axis won't work these times), other times as "Generic" which works for steam games (Rivals of Aether and Rushdown Revolt) but only until it gets unplugged. No idea what I am doing wrong, but it is maddening.

Are you sure you don't trigger an input mode change by mistake when playing?
If so you probably can avoid the problem by disabling/redefining mode selection shortcuts in mode_selection.hpp

When unplugged/Replugged the LBX seems to go back to its default key mapping

**Comment by scMarth** [comment]
For me LBX with FGC mode, it doesn't make me re-map my buttons everytime. It works well as a hitbox for fighting games. I wish there was a mode that let me map all the other buttons on steam though
